File: src/targets/javascript/fetch/fixtures/postdata-malformed.js
const url = 'https://httpbin.org/anything';
const options = {method: 'POST', headers: {'content-type': 'application/json'}};

try {
  const response = await fetch(url, options);
  const data = await response.json();
  console.log(data);
} catch (error) {
  console.error(error);
}

File: src/targets/rust/reqwest/fixtures/postdata-malformed.rs
use reqwest;

#[tokio::main]
pub async fn main() {
    let url = "https://httpbin.org/anything";

    let client = reqwest::Client::new();
    let response = client.post(url)
        .header("content-type", "application/json")
        .send()
        .await;

    let results = response.unwrap()
        .json::<serde_json::Value>()
        .await
        .unwrap();

    println!("{}", results);
}
